### PR TITLE
Update graph graphics

### PR DESCRIPTION
**Pull request by lubej**

### CODE REVIEW

File: src/app/pages/HomePage/Graph/Graph/index.tsx [hunk 1]
```diff
@@ -168,216 +172,184 @@ const GraphCmp: ForwardRefRenderFunction<SVGSVGElement, GraphProps> = (
...
+            fill={graphTheme.text}
           />
-          <text x="150.5" y="39" fill={graphTheme.text} fontSize="4px">
```
Comment (buberdds) [inline src/app/pages/HomePage/Graph/Graph/index.tsx:265]: All these svg paths that renders labels look blurry. Should we keep text nodes?

**Review by buberdds — COMMENTED** [review]
1) ParaTimes circles are semi transparent and connecting lines are visible. 

2) Why Sapphire and Cipher look like they are enabled, but Emerald don't. This is more question to Don.

File: src/app/pages/HomePage/Graph/Graph/index.tsx (hunk 1 above)
Comment (lubej) [reply src/app/pages/HomePage/Graph/Graph/index.tsx:265]: Wanted to do this in next PR, as there are new animations, which affect the fonts as well. To keep PRs manageable.

Comment (buberdds) [reply src/app/pages/HomePage/Graph/Graph/index.tsx:265]: ah Ok wasn't aware that we will have follow up updates 👍

**Review by buberdds — APPROVED** [review]

**Pull request merged**